### PR TITLE
{math}[foss/2020b] Demystify v0.0.17 w/ Python 3.8.6

### PR DESCRIPTION
**Pull request by lexming**
(created using `eb --new-pr`)

Depends on:
* [x] https://github.com/easybuilders/easybuild-easyconfigs/pull/18284

### CODE REVIEW

**Review by verdurin — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/d/Demystify/Demystify-0.0.17-foss-2020b.eb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+source_urls = [PYPI_LOWER_SOURCE]
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['93779a131aa78f7de0dffdbe8e89124836b5eb6d0e9b4bc385821845a4628f52']
+
+dependencies = [
+    ('Python', '3.8.6'),
+    ('PySAT', '0.1.7.dev1'),
+    ('SciPy-bundle', '2020.11'),
+    ('Z3', '4.8.10', '-Python-%(pyver)s'),
```
[the remark this answers is not in the document]
Comment (lexming) [reply easybuild/easyconfigs/d/Demystify/Demystify-0.0.17-foss-2020b.eb]: Thanks @verdurin, fixed in https://github.com/easybuilders/easybuild-easyconfigs/pull/17990/commits/b7f92d85ea82cdf1c76c705716699b19ad470b42

**Review by verdurin — DISMISSED** [review]
Looks fine.

**Comment by verdurin** [comment]
@boegelbot please test @ generoso

**Comment by verdurin** [comment]
Test report by @verdurin
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in total)
easybuild-c7.novalocal - Linux CentOS Linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/verdurin/cd77f813e87b4a568d3aadf67c4088f6 for a full test report.

**Comment by lexming** [comment]
@boegelbot: please test @ generoso

**Comment by lexming** [comment]
Failed tests are due to missing `z3_solver`. This PR needs #18284

**Comment by lexming** [comment]
@boegelbot: please test @ generoso

**Comment by lexming** [comment]
@boegelbot: please test @ generoso

**Comment by lexming** [comment]
@boegelbot: please test @ generoso

**Comment by lexming** [comment]
@boegelbot please test @ jsc-zen2

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
zen2-rockylinux-89 - Linux Rocky Linux 8.9, x86_64, AMD EPYC 7452 32-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/SebastianAchilles/ea80c98dbd502363d91eefd9baf1b4ea for a full test report.

**Review by SebastianAchilles — APPROVED** [review]
lgtm

**Pull request merged**